source/localizable/smart_cart/orders_api.html.md.erb

```diff
@@ -158,6 +158,61 @@ also get a `422` HTTP status code (Unprocessable Entity), with the following bod
 
 [More on API error handling](/api/v3/#error-handling)
 
+## Test requests (sandbox)
+
+The following d​emo order codes​ are available on production for testing reasons, andrepresent different order states,
+trying to cover all possible responses.
+
+#### DEMO-OPEN
+
+A new order that is not accepted yet, and has no voucher
```

Comment:
Lets refer to "voucher" as "courier voucher". WDYT? 

(also, missing period `.` 😺 )